packages/terra-date-input/src/DateInput.jsx

```diff
@@ -102,6 +102,8 @@ const propTypes = {
    * Custom input attributes to apply to the year input
    */
   yearAttributes: PropTypes.object,
+  errorMessageId: PropTypes.string,
+  ariadescribedby: PropTypes.string,
```

Comment:
we need to add Prop description for above two props !!

Reply:
These are removed as per latest changes.